.github/workflows/version-notes-status-check.yml

```
@@ -82,6 +82,6 @@ jobs:
         deno `
           run `
           --allow-read `
-          "${{ github.workspace }}/.github/cicd/validate-notes.ts" `
+          "${{ github.workspace }}https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-notes.ts`
```

Remove the `${{ github.workspace }}` variable.  This is a path and for paths. But now we are using URLs.
Don't forget the double quote in the beginning.  You removed the one on the right end when you entered the URL only leaving one behind without a matching pair.

```suggestion
          https://raw.githubusercontent.com/KinsonDigital/Infrastructure/v3.0.3/cicd/validate-notes.ts`
```